en/api/action.md

```diff
@@ -41,7 +41,7 @@ Chart actions supported by ECharts are triggered through [dispatchAction](~echar
 Highlights specified data graphics.
 
 ```ts
-// If highlight series:
+// If highlight series（in the `sunburst` series,`highlight` has been deprecated since `v5.0.0`, please use `sunburstHighlight` instead）：

```

Comment:
Please revert this change.

Reply:
I have  reverted it , please review again